src/arx7.js

```diff
@@ -64,8 +64,60 @@ client.addListener('message', (from, to, text, message) => {
 
   // Queries
   if (to === client.nick) {
-    log(`Query from ${from}: ${text}`);
-    client.say(from, "I'm a bot! Contact Desch, Jukey, or Aoi-chan for help");
+    // Check for Admin Queries
+    if (config.admins.includes(from)) {
+      log(`Admin Query from ${from}: ${text}`);
+      let usage = `[add|remove] [plugin] [channel] [password]`;
+
+      if (text.startsWith('add ') || text.startsWith('remove ')) {
+        let args = text.split(' ');
+
+        if (args.length != 4) {
+          log(`Invalid command count`);
```

Comment:
command count? Drop count, maybe? idk sounds like the number of commands you gave is wrong.